src/jit.rs

```diff
@@ -1033,6 +1026,63 @@ impl<'a, C: ContextObject> JitCompiler<'a, C> {
         self.emit_ins(X86Instruction::mov(OperandSize::S64, R11, host_addr));
     }
 
+    fn emit_memory_load(&mut self, dst: u8, vm_addr: Value, len: u64) {
```

Comment:
`emit_memory_load()` and `emit_memory_store()` are not needed because the current design of `emit_address_translation()` already differentiates `AccessType::`Load and `AccessType::Store` and the width of the access.

Reply:
Done